src/tracking.js

```diff
@@ -203,7 +188,7 @@ export function setupLogger() {
       dimensions: {
         components: getComponents().join(","),
         integrationSource,
-        isPayPalDomain: isLoadedInFrame,
+        isPayPalDomain: Boolean(isLoadedInFrame),
```

Comment:
I'm not sure if this is still the case but we've seen in the past signalFX dropping boolean values. Can we make sure this is a true/false string or some other string value?

Reply:
Good observation! I wasn't aware of that - I'll change this to a string